src/System.Windows.Forms.Primitives/src/Windows/Win32/PInvoke.SHAutoComplete.cs

```diff
@@ -2,23 +2,18 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Runtime.InteropServices;
-
-internal static partial class Interop
+namespace Windows.Win32
 {
-    internal static partial class Shlwapi
+    internal static partial class PInvoke
     {
-        [DllImport(Libraries.Shlwapi, ExactSpelling = true)]
-        public static extern HRESULT SHAutoComplete(IntPtr hwndEdit, SHACF flags);
-
-        public static HRESULT SHAutoComplete(IHandle hwndEdit, SHACF flags)
+        public static HRESULT SHAutoComplete(IHandle<HWND> hwndEdit, SHELL_AUTOCOMPLETE_FLAGS flags)
         {
             HRESULT result = SHAutoComplete(hwndEdit.Handle, flags);
             GC.KeepAlive(hwndEdit);
             return result;
```

Comment:
We want to use the following existing pattern: 

```suggestion
        public static HRESULT SHAutoComplete<T>(in T hwndEdit, SHELL_AUTOCOMPLETE_FLAGS flags) where T : IHandle<HWND>
        {
            HRESULT result = SHAutoComplete(hwndEdit.Handle, flags);
            GC.KeepAlive(hwndEdit.Wrapper);
            return result;
```